### PR TITLE
chore: add OpenShift 4.9 to supported platforms

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]
LGTM, but please make mdl happy :)

File: deploy/docs/Best_Practices.md [hunk 1]
```diff
@@ -645,7 +649,7 @@ fluentd:
 ```
 
 It is going to replace all `Password: <pass>` occurence in logs with `Password: ***` and replace namespace to `REDACTED`
-for [all containers](#difference-between-varlogpods-and-varlogcontainers) named `test-container`.
+for [all containers][#difference-between-varlogpods-and-varlogcontainers] named `test-container`.
```
Comment (sumo-drosiek) [inline deploy/docs/Best_Practices.md:652]:
```suggestion
for [all containers](#difference-between-varlogpods-and-varlogcontainers) named `test-container`.
```

**Comment by kkujawa-sumo** [comment]
> oh wow, i wonder why markdownlint didn't complained before about those changes 😕

Maybe new version of markdownlint...

File: deploy/docs/existingPrometheusDoc.md [hunk 2]
```diff
@@ -27,6 +28,17 @@ If you do not wish to modify your Prometheus Operator and wish to run it side-by
 with our collection, please refer to our
 [How to install our Prometheus side by side with your existing Prometheus](./SideBySidePrometheus.md) documentation.
 
+## Requirements
+
+If you don’t already have a Sumo account, you can create one by clicking the Free Trial button on https://www.sumologic.com/.
+
+The following are required to setup Sumo Logic's Kubernetes collection.
+
+- An [Access ID and Access Key](https://help.sumologic.com/Manage/Security/Access-Keys) with [Manage Collectors](https://help.sumologic.com/Manage/Users-and-Roles/Manage-Roles/05-Role-Capabilities#data-management) capability.
```
Comment (sumo-drosiek) [inline deploy/docs/existingPrometheusDoc.md:37]: quite long line

**Review by sumo-drosiek — APPROVED** [review]
:rocket:

**Pull request merged**